redbot/cogs/mod/names.py

```diff
@@ -112,17 +191,18 @@ class ModInfo(MixinMeta):
         created_on = _("{}\n({} days ago)").format(user_created, since_created)
         joined_on = _("{}\n({} days ago)").format(user_joined, since_joined)
 
+        if user.status.name == "online":
+            statusemoji = "\N{LARGE GREEN CIRCLE}"
+        elif user.status.name == "offline":
+            statusemoji = "\N{MEDIUM WHITE CIRCLE}"
+        elif user.status.name == "dnd":
+            statusemoji = "\N{LARGE RED CIRCLE}"
+        elif user.status.name == "streaming":
+            statusemoji = "\N{LARGE PURPLE CIRCLE}"
+        elif user.status.name == "idle":
+            statusemoji = "\N{LARGE ORANGE CIRCLE}"
```

Comment:
Streaming status isn't actually a status so we gotta check the activities to know if we should put purple circle.

```suggestion
        if any(a.type is discord.ActivityType.streaming for a in user.activities):
            statusemoji = "\N{LARGE PURPLE CIRCLE}"
        elif user.status.name == "online":
            statusemoji = "\N{LARGE GREEN CIRCLE}"
        elif user.status.name == "offline":
            statusemoji = "\N{MEDIUM WHITE CIRCLE}"
        elif user.status.name == "dnd":
            statusemoji = "\N{LARGE RED CIRCLE}"
        elif user.status.name == "idle":
            statusemoji = "\N{LARGE ORANGE CIRCLE}"
```

// I don't know how I didn't add this comment to review 🤔